lnencrypt/tor.go

```diff
@@ -1,76 +0,0 @@
-package lnencrypt
```

Comment:
This should probably be removed from the first commit if it's going to be deleted again in the next one.